distro/docker/src/main/docker/Dockerfile.jvm

```diff
@@ -1,9 +1,12 @@
-FROM fabric8/java-alpine-openjdk11-jre
+FROM registry.access.redhat.com/ubi8/openjdk-11:latest
 
 ENV APP_URL="${docker.app.file}"
 ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 ENV AB_ENABLED=jmx_exporter
+ENV JAVA_APP_DIR=/deployments
 
-ADD "$APP_URL" /deployments
+EXPOSE 8080 8778 9779
```

Comment:
What are the 8778 and 9779 ports used for?

Reply:
umm I am not sure! I used the dockerfile we use for productization and I simply copied it

Reply:
OK we can keep it as-is for now.  

@Apicurio/developers any idea what those two additional ports are for?  I'm thinking 8778 might be the debug port.  But I have no idea what 9779 would be.